### ISSUE TITLE
Alt-dragging sometimes mutates other arrows

### ISSUE THREAD

**Issue by gaearon** [excalidraw/excalidraw#739]
I can't reproduce this consistently but this bug has been a huge time-sink.

Basically, if I draw some arrows, resize them, and then Alt+Drag to copy them, at some point all previous versions of an arrow might "mutate" to match one of them. So I have to go around and fix every single copy.

I don't have a consistent repro unfortunately.

**Comment by vjeux** [comment]
duplicateElement is doing a shallow copy: https://github.com/excalidraw/excalidraw/blob/1e4ce776127a4cc4f710c3dc134c684e65232916/src/element/newElement.ts#L66

It used to be okay since the element was flat. But for multi-point arrows, `points` is an array: https://github.com/excalidraw/excalidraw/blob/1e4ce776127a4cc4f710c3dc134c684e65232916/src/element/newElement.ts#L38

So the current behavior is to share the same points array for the cloned element. We need to do a deeper copy.

**Comment by vjeux** [comment]
Repro case:
- Create a 3-point arrow
- Alt drag the arrow somewhere else
- Resize the copy
- Refresh the page
- The initial arrow is resized to match the copy

**Comment by vjeux** [comment]
I'm not really sure what's going on, I changed `duplicateElement` to do a deep copy of the points array, but the bug still occurs.

```js
  const copy = { ...element, points: [...element.points] };
```

I checked that `duplicateElement` is being called and that the elements.points objects are indeed different. But it still resizes on reload. We must be doing something weird with resizing.

cc @GasimGasimzada, @dwelle
